shared/test/test_deliverer.rb

```diff
@@ -0,0 +1,48 @@
+require_relative 'test_helper'
```

Comment:
Why'd you put a test for a class in `lib/cdo` in `shared/test` instead of `lib/test/cdo`?

Reply:
oh, that's a much better location. I just looked for where the existing `Poste` tests were